requirements/test.txt

```diff
@@ -3,6 +3,8 @@
 codecov
 coverage
 flake8
+nose
```

Comment:
Sir when i testing the test code inside the python terminal there it told that it need nose so i installed and the code started running.
Here is a screenshot of the error.
![1](https://user-images.githubusercontent.com/35897183/75099196-ddc80880-55e4-11ea-8a1a-6c051c4c8128.PNG)
That's why i have added it.

Reply:
ok, Thanks for the notice, we should not use `nose`, this is a bug that you can resolve. 
I suppose it happens because of your numpy version. What are your Numpy version, DIPY version, VTK version, fury version?

Reply:
This is the screen-shot here.
![3](https://user-images.githubusercontent.com/35897183/75099468-0a315400-55e8-11ea-9ab4-2a0f05870996.PNG)